src/main/resources/assets/styles/styles.scss

```diff
@@ -75,6 +75,18 @@
         width: 100%;
     }
 
+    #select-type {
+        .dropdown-grid .viewer {
+            padding-left: 20px;
+
+            &.non-selectable {
```

Comment:
Bold the groups so they are more visible then the rest? I don't know about the opacity addition.

Reply:
The PR was in draft as it wasn't ready for review yet. It is now :)